dataland-backend/src/main/kotlin/org/dataland/datalandbackend/model/sfdr/categories/general/subcategories/SfdrGeneralGeneral.kt

```diff
@@ -23,8 +22,11 @@ data class SfdrGeneralGeneral(
     @field:JsonProperty(required = true)
     val fiscalYearEnd: LocalDate,
 
-    @field:Schema(example = JsonExampleFormattingConstants.REFERENCED_REPORTS_DEFAULT_VALUE)
-    override val referencedReports: Map<String, CompanyReport>? = null,
+    @field:Schema(
+        implementation = Map::class,
+        example = JsonExampleFormattingConstants.REFERENCED_REPORTS_DEFAULT_VALUE,
+    )
+    val referencedReports: Map<String, CompanyReport>? = null,
 
     val scopeOfEntities: YesNoNa? = null,
-) : FrameworkBase
```

Comment:
:FrameworkBase should be restored, as well es the override val referencedReports as the documents interface should be used here.

Reply:
done.

Reply:
ok